projects/storefrontapp/src/.well-known/apple-developer-merchantid-domain-association.txt

```diff
@@ -0,0 +1,74 @@
+MIIQdQYJKoZIhvcNAQcCoIIQZjCCEGICAQExCzAJBgUrDgMCGgUAMIGDBgkqhkiG9w0BBwGgdgR0
```

Comment:
Not sure how the apple pay works, but this seems to be some hashed value. Is it the file that we should keep in our repo, or in other words is it any kind of sensitive data?

Reply:
yep, mistake. this file must be added by the admin.